client/src/utils/getWeb3.js

```diff
@@ -1,38 +1,45 @@
 import Web3 from "web3";
 
+function findWeb3() {
+  return new Promise((resolve, reject) => {
+    // Modern dapp browsers...
+    if (window.ethereum) {
+      const web3 = new Web3(window.ethereum);
+      try {
+        // Request account access if needed.
+        // See details here: https://medium.com/metamask/https-medium-com-metamask-breaking-change-injecting-web3-7722797916a8
+        window.ethereum.enable().then(() => resolve(web3));
+      } catch (error) {
+        reject(error);
+      }
+    }
+    // Legacy dapp browsers...
+    else if (window.web3) {
+      // Use Mist/MetaMask's provider.
+      const web3 = window.web3;
+      console.log("Injected web3 detected.");
+      resolve(web3);
+    }
+    // Fallback to localhost; use dev console port by default...
+    else {
```

Comment:
Actually maybe this `else` case should only be done if the "load" event is being triggered?